.docker/entrypoint/spec/jekyll_commander_spec.rb

```diff
@@ -73,4 +74,42 @@
       end
     end
   end
+
+  describe '#execute :serve' do
+    describe '_site' do
+      data_dir = File.join(__dir__, '..', '..', '..', 'tests', 'full')
+      @thread;
```

Comment:
Codacy found an issue: [Variable `@thread` used in void context.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5780130)